courses/free-courses-en.md

```diff
@@ -420,6 +421,7 @@
 * [Programming Foundations with Javascript, HTML and CSS](https://www.coursera.org/learn/duke-programming-web) - Owen Astrachan, Robert Duvall, Andrew D. Hilton, Susan H. Rodger (Coursera)
 
 
+
```

Comment:
```suggestion
```

You just added one too many black lines.

Reply:
Thanks 
I've already fix that